.github/workflows/doc-lint.yml

```diff
@@ -34,3 +34,15 @@ jobs:
             echo "You need to run ./utils/fix-zh-doc-segment.py to fix them."
             exit 1
           fi
+
+  Chinse-Copywriting-lint:
+    name: Chinese Copywriting
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+    steps:
+      - uses: actions/checkout@v3
+      - name: check Chinese copywriting
+        run: |
+          curl -sSL https://git.io/JcGER | bash
+          autocorrect --version
```

Comment:
> Also, could we use a submodule to execute this CI for security considerations?

https://github.com/apache/apisix/pull/6340#issuecomment-1043880615 No suggestions on submodule.

Reply:
Check this https://github.com/apache/apisix/issues/6528#issuecomment-1062579469

Reply:
> Check this [#6528 (comment)](https://github.com/apache/apisix/issues/6528#issuecomment-1062579469)

I checked this. 

Above all, action as submodule had different opinions, we have to wait for other advices.

Reply:
see https://github.com/apache/apisix-dashboard/tree/master/.github/actions

We're using submodule in dashboard project 😅

Reply:
> Hi, will this change PR's contents?

No, it won't change PR's content.

Reply:
According to ASF's policies, we have to use a submodule instead of cURL a link, bcoz we can't control the link and its service :(

Reply:
cc @spacewander @moonming  to have a confirmation.

Reply:
> According to ASF's policies, we have to use a submodule instead of cURL a link, bcoz we can't control the link and its service :(

Thx, I known it. We should indicate which is the acceptable. The target download link or the submodule?

Note it: local action equals to download link. Autocorrect no supports for all kinds of package managers.

Reply:
I'm only concerned about whether the link `https://git.io/JcGER` will contain not good codes? 😄 Because it's not versioned (that's why the ASF asks to use a specified submodule instead).

cc @bzp2010 @Yiyiyimu to have a confirmation. If it's good to go, then let't merge this PR with honor! 😄

Reply:
That's good!

Reply:
It had been changed.

Reply:
versioned link LGTM

Reply:
Hi @kwanhur, significant changes! 🎉

I just reviewed what apisix-dashboard does, the submodule uses a specific commit id, but `get_latest_release` will always get the latest releases. If this repo releases a new version with unexpected codes, it will not look good for us either.

Hope you can understand and use a specific version/commit instead of the latest. ❤️

![image](https://user-images.githubusercontent.com/2106987/159632416-6575272f-bcae-456e-bd39-46f1d28f787b.png)

Reply:
Had specified fixed version.